### ISSUE TITLE
VI_V_SYNC_REG too big

### ISSUE THREAD

**Issue by Frank-74** [gonetz/GLideN64#1949]
Getting lots of "VI_V_SYNC_REG too big"'s written into log for all games.

Using latest WIP, default settings.  Intel HD Graphics.

**Comment by loganmc10** [comment]
I don't think that value is modified by the GFX plugin, what emulator are you using?

**Comment by Frank-74** [comment]
Project64, it's written into gliden64.log, so it must be coming from GLideN64.

**Comment by loganmc10** [comment]
This must be happening only in LLE is that correct?

**Comment by Frank-74** [comment]
Using HLE.  Just double checked.  Deleted gliden64.log,  ran with HLE, lots of VI_V_SYNC_REG lines in gliden64.log.  Deleted again, ran with LLE, still lots of VI_V_SYNC_REG lines.

**Comment by Kidboy17** [comment]
I also found that in my gliden.log but I don't have any problems using VSync, VI's stay around 60.

**Comment by Frank-74** [comment]
@Kidboy17 Did you restart the game after enabling VSync?  There's no change enabling with the game running here.  I have to restart the rom.

**Comment by loganmc10** [comment]
I tested in mupen64plus and I don't see any of that in gliden64.log

You should try with an older version of PJ64. The GFX plugin gets the value of VI_V_SYNC_REG from the emulator

**Comment by Frank-74** [comment]
Thanks, I'll open the issue at Project64's Github.

**Comment by Frank-74** [comment]
Re-opened because it's not a problem with Project64.  Discussed here: https://github.com/project64/project64/issues/1505

It writes 15 lines of VI_V_SYNC_REG too big on startup of ROM, then no more.
It only writes the VI_V_SYNC_REG too big line while VI_V_SYNC_REG is 0.

**Comment by Kidboy17** [comment]
@Frank-74 I've used VSync since end of March and haven't had any issues with it. I always have VSync, N64 style 3 point and Render 2D enabled unless I am looking for bugs.

**Comment by loganmc10** [comment]
This has nothing to do with the Vsync setting, this is an internal register used by the N64, it's unrelated to your computer's Vsync setting

**Comment by clobber** [comment]
I had a lot of these lines spamming my gliden64.log too, but I should note I'm using my own core port of M64P in the latest (but fairly old at this point now) stable 2.5 release. GLideN64 build @ 9804fb69dd398943a0d105942c4e210926e89319

If we want to know why, perhaps we should investigate the actual log being spit out from the code in the first place: https://github.com/gonetz/GLideN64/blob/68a2c939b4f3a55a5095be8366010ded0c0562c2/src/FrameBuffer.cpp#L1083-L1087

Maybe it's left in by mistake?

**Comment by loganmc10** [comment]
Current versions of mupen64plus don't trigger VI interrupts if VI_V_SYNC_REG=0, so it works differently than PJ64 or older versions of mupen64plus.

The documents say that a VI interrupts is triggered when VI_V_SYNC_REG == VI_V_INTR_REG, which isn't possible when VI_V_SYNC_REG is 0. @project64 might be able to comment on how PJ64 works in regards to VI interrupts.

Basically, yes, the error message is probably inaccurate when it says that the value is too big, but it is likely correct that the value of VI_V_SYNC_REG is wrong/impossible
